src/signal.cr

```diff
@@ -117,6 +117,21 @@ enum Signal : Int32
     Crystal::System::Signal.trap(self, handler)
   end
 
+  # Returns any existing handler for this signal
+  #
+  # ```
+  # Signal::USR1.trap { }
+  # existing = Signal::USR1.trap_handler?
+  #
+  # Signal::USR1.trap do |signal|
+  #   existing.try &.call(signal)
+  #   # ...
+  # end
+  # ```
+  def trap_handler?
+    Crystal::System::Signal.trap_handler?(self)
```

Comment:
Do I understand it correctly that `Signal::CHLD.trap_handler?` will always return `nil`?

Reply:
Yeah, I suppose we'll need to account for the special handling of SIGCHLD here.